src/libcore/cell.rs

```diff
@@ -543,6 +543,59 @@ impl<T> RefCell<T> {
         debug_assert!(self.borrow.get() == UNUSED);
         unsafe { self.value.into_inner() }
     }
+
+    /// Replaces the wrapped value with a new one, returning the old value,
```

Comment:
I wonder if the docs should mention that these are simply convenience functions for their one-liner implementations? I wouldn't want people thinking that they're more "magical" than they really are. Thoughts @steveklabnik? Not sure if we do something similar elsewhere.